include/aspect/postprocess/tracer.h

```diff
@@ -35,36 +38,125 @@ namespace aspect
     template <int dim>
     class PassiveTracers : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
```

Comment:
Not your fault, but could you add a brief documentation to what this class does?